controls/def.cf

```diff
@@ -246,6 +246,10 @@ bundle common def
                           "true",
                           "false");
 
+      "control_common_lastseenexpireafter" -> { "ENT-10414" }
+        string => "10080", # 10080 minutes is 1 week
```

Comment:
Be nice to mention that this is also the default in the binaries right?

https://docs.cfengine.com/docs/3.21/reference-components.html#lastseenexpireafter

Reply:
Sure, can do.

Reply:
Done.